.gitignore

```diff
@@ -30,3 +30,7 @@ override.tf.json
 
 # Include tfplan files to ignore the plan output of command: terraform plan -out=tfplan
 # example: *tfplan*
+
+# Ignore Release components
+goreleaser*
```

Comment:
the reason i specified goreleaser is that the download and install of this would add untracked files in the dir which goreleaser does not like and would fail to start the build/release.